src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/CompilerPass/RegisterEventListenersAndSubscribersPassTest.php

```diff
@@ -114,6 +114,8 @@ public function testProcessEventListenersWithMultipleConnections()
     {
         $container = $this->createBuilder(true);
 
+        $container->setParameter('connection_param', 'second');
```

Comment:
IMO, we should add an assertion about the resolved value somewhere.

Reply:
We do not. 
On line 142-148 service `d` for connecion `%connection_param%` is created.  
On line 174 event manager for `second` connection is taken.
On line 179 new event listener is added.
On line 186-193 events for connection `second` is checked.

So if resolved value is not `second` hen assertion from line 186-193 will fail.

Reply:
I'm not familiarized with this test. If the new behavior is covered as described, it's fine for me +1